next.config.js

```diff
@@ -14,4 +14,16 @@ const nextConfig = {
   },
 };
 
+// For building on vercel: https://github.com/Automattic/node-canvas/issues/1779
+if (
+  process.env.LD_LIBRARY_PATH == null ||
+  !process.env.LD_LIBRARY_PATH.includes(
+    `${process.env.PWD}/node_modules/canvas/build/Release:`,
+  )
+) {
+  process.env.LD_LIBRARY_PATH = `${
+    process.env.PWD
+  }/node_modules/canvas/build/Release:${process.env.LD_LIBRARY_PATH || ''}`;
+}
+
```

Comment:
seems like this is not needed anymore, right? because `jsdom-napi-rs-canvas` is being used?